amplify/backend/function/expressServer/src/services/stockRoutesServices.js

```diff
@@ -89,11 +90,43 @@ function getStockPriceData (stocks) {
 
 };
 
-const getStockSummary =  (schema) => {
+// This function makes an API call to the API gateway for the recommender lambda function
+const getRecomms = async (stock) => {
+    var axios = require('axios').default;
```

Comment:
Each time this function runs you will keep requiring it again, this function is run anytime user visits stock discovery, I'd require it up the top of the file globally so only executes once.

Reply:
Good spot apologies for that, I noticed this last night and meant to change it but it skipped my mind. Will change this now.